examples/de.dwd.globalcache.json

```diff
@@ -0,0 +1,128 @@
+{
+    "id": "urn:x-wmo:md:deu:dwd-offenbach:global-cache-service",
+    "type": "Feature",
+    "conformsTo": [
+        "http://wis.wmo.int/spec/wcmp/2.0"
+    ],
+    "time": {
```

Comment:
Thinking out loud, do/should we specify a time for a service, or do we advocate for `null` as a rule?